docs/web-ui.md

```diff
@@ -94,9 +94,76 @@ This page displays the details of a specific job identified by its job ID.
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail3.png" title="Stages detail" alt="Stages detail">
+</p>
+
+### Stage detail
+The stage detail page begins with information like total time across all tasks, [Locality level summary](tuning.html#data-locality), [Shuffle Read Size / Records](rdd-programming-guide.html#shuffle-operations) and Associated Job IDs.
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail4.png" title="Stage header" alt="Stage header" width="30%">
+</p>
+
+There is also a visual representation of the directed acyclic graph (DAG) of this stage, where vertices represent the RDDs or DataFrames and the edges represent an operation to be applied.
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail5.png" title="Stage DAG" alt="Stage DAG" width="50%">
+</p>
+
+Summary metrics for all task are represented in a table and in a timeline.
+* **[Tasks deserialization time](configuration.html#compression-and-serialization)**
+* **Duration of tasks**.
+* **GC time** is the total JVM garbage collection time.
+* **Result serialization time** is the time spent serializing the task result on a executor before sending it back to the driver.
+* **Getting result time** is the time that the driver spends fetching task results from workers.
+* **Scheduler delay** is the time the task waits to be scheduled for execution.
+* **Peak execution memory** it's the maximum memory used by the internal data structures created during shuffles, aggregations and joins.
+* **Shuffle Read Size / Records**.Total shuffle bytes read, includes both data read locally and data read from remote executors.
+* **Shuffle Read Blocked Time** is the time that tasks spent blocked waiting for shuffle data to be read from remote machines.
+* **Shuffle Remote Reads** is the total shuffle bytes read from remote executors.
+* **Shuffle spill (memory)** is the size of the deserialized form of the shuffled data in memory.
+* **Shuffle spill (disk)** is the size of the serialized form of the data on disk.
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail6.png" title="Stages metrics" alt="Stages metrics">
+</p>
+
+Aggregated metrics by executor show the same information aggregated by executor
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail7.png" title="Stages metrics per executor" alt="Stages metrics per executors">
+</p>
+
+**Accumulators** are a type of shared variables, it provide a mutable variable that can be updated inside of a variety of transformations. It is possible to create accumulators with and without name, but only named accumulators are displayed.
+
+<p style="text-align: center;">
+  <img src="img/AllStagesPageDetail8.png" title="Stage accumulator" alt="Stage accumulator">
+</p>
+
+Tasks details show the information of every task
```

Comment:
Period at the end. Can we say any more, like the types of information shown? Just a sentence.